protocol/synthetix/test/integration/modules/core/MarketManagerModule.test.ts

```diff
@@ -289,6 +209,7 @@ describe('MarketManagerModule', function () {
 
       describe('withdraw some from the market', async () => {
         before(withdrawRestore);
+        
```

Comment:
I have pushed local branch `sip-365-remove-core-fee-switch` to the repo so CI will actually run. Will keep it up to date, keep an eye of failed workflows - they are legit now.

Reply:
<img width="847" alt="image" src="https://github.com/Synthetixio/synthetix-v3/assets/28145325/bdc46652-7761-414a-9d91-55f326ee19bc">

Reply:
Fixed.